packages/diagrams/frontend/sirius-components-diagrams/src/renderer/panel/experimental-svg-export/RectElementSVGExportHandler.ts

```diff
@@ -35,6 +35,14 @@ export class RectElementSVGExportHandler implements IElementSVGExportHandler {
       rect.setAttribute('rx', String(style.borderRadius));
       rect.setAttribute('fill', style.backgroundColor);
 
+      if (style.backgroundColor.endsWith(', 0)')) {
```

Comment:
Looks a little brittle, but I guess it's OK for an experimental version.

Reply:
Updated